x-pack/plugins/maps/server/mvt/mvt_routes.ts

```diff
@@ -10,6 +10,8 @@ import { schema } from '@kbn/config-schema';
 import { KibanaRequest, KibanaResponseFactory, Logger } from 'src/core/server';
 import { IRouter } from 'src/core/server';
 import type { DataRequestHandlerContext } from 'src/plugins/data/server';
+// @ts-ignore not typed
+import { AbortController } from 'abortcontroller-polyfill/dist/cjs-ponyfill';
```

Comment:
recommended way of importing the polyfill. Node will start supporting `AbortController` starting v15. cc @lukasolson